sass/_Variables.scss

```diff
@@ -24,13 +24,14 @@ $color-light-grey: #bcc3ca;
 $color-blueish-white-grey: #e6ecf0;
 $color-blueish-white: #f7f8f9;
 $color-light-greyish-blue: #9cb4cb;
-$color-disabled-grey: #ddd;
 // Coveo Partners Colors
 $color-coveo-for-sitecore: #dc291e;
 $color-coveo-for-salesforce: #009ddc;
 $color-green: #4caf50;
-$color-dark-green: hsl(122, 39%, 37%);
+$color-dark-green: #3a833c;
 $color-red: #cc0d00;
+$color-orange: #c65306;
```

Comment:
The color from the design is `#f28134`, but its contrast ratio against white is way too low:`2.63:1`, under the minimum `4.5:1`.
`#c65306` has a contrast ratio against white of `4.52:1`.